src/cogserver/util.py

```diff
@@ -41,3 +45,13 @@ def replace_dependency(app=None, new_dependency=None, arg_name=None):
                         r.dependencies.extend([depends])
             #print([[e.call for n in e.query_params if n.name == arg_name] for e in r.dependant.dependencies if e])
             #print(r.dependencies)
+
+
+
+
+if __name__ == "__main__":
+    asyncio.run(create_vrt_from_urls(
+        urls=[
+            "https://undpgeohub.blob.core.windows.net/geo-nightlights/test/SVDNB_npp_d20231229.rade9d_sunfiltered.tif",
+            "https://undpgeohub.blob.core.windows.net/geo-nightlights/test/SVDNB_npp_d20240101.rade9d_sunfiltered.tif"
+        ]))
```

Comment:
Looks like `create_vrt_from_urls` function is no longer in this `util.py`. You should remove this code.

For testing, you maybe can introduce a testing library such as pytest, for example. create a test file like `test_utils.py`.